events/elections/2017/quintonhoole_bio.md

```diff
@@ -28,17 +28,17 @@ I have an M.Sc in Computer Science.
 # Highlights of What I've done on Kubernetes thus far
 
 Initially I led the effort to get our CI Testing stuff initiated and
-[working effectively](https://github.com/kubernetes/community/blob/master/contributors/devel/writing-good-e2e-tests.md), which was critical to being able to launch v1.0 successfully,
+[working effectively](https://git.k8s.io/community/contributors/devel/writing-good-e2e-tests.md), which was critical to being able to launch v1.0 successfully,
 way back in July 2015.  Around that time I handed it over to the now-famous SIG-Testing.
 
 After that I initiated and led the [Cluster
-Federation](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/federation/federation.md)
+Federation](https://git.k8s.io/community/contributors/design-proposals/federation/federation.md)
```

Comment:
Strangely, this link does not work either.

Reply:
This one's a bug.  My tree was in a stale state when I found the dead link.  That file's moved at least twice since that link was last correct.  it's now https://git.k8s.io/community/contributors/design-proposals/multicluster/federation.md